DOCUMENTATION.md

```diff
@@ -296,94 +296,55 @@ These configuration elements are enough for most connection scenarios. Now we wi
-let config = new Config.ClientConfig();
-Client.newHazelcastClient(config).then(function(client) {
-    // Some operations
-});
-```
-
-**Declarative Configuration:**
+const { Client } = require('hazelcast-client');
 
-Hazelcast Node.js client looks for a `hazelcast-client.json` in the current working directory unless you provide a configuration object
-at the startup. If you intend to configure your client using a configuration file, then place a `hazelcast-client.json` in the directory
-of your application's entry point.
+// ...
 
-If you prefer to keep your `hazelcast-client.json` file somewhere else, you can override the environment variable `HAZELCAST_CLIENT_CONFIG`
-with the location of your config file. In this case, the client uses the configuration file specified in the environment variable.
+const client = await Client.newHazelcastClient({
+    clusterName: 'name of your cluster'
+});
+// Some operations
+```
 
-For the structure of `hazelcast-client.json`, see the [hazelcast-client-full.json file](hazelcast-client-full.json). You
-can use only the relevant parts of the file in your `hazelcast-client.json` and remove the rest. The default configuration is used for any
-part you do not explicitly set in the `hazelcast-client.json` file.
+It's also possible to omit the config object in order to use the default settings.
 
----
+```javascript
+const client = await Client.newHazelcastClient();
+// Some operations
+```
 
-If you run the Hazelcast IMDG members in a different server than the client, you most probably have configured the members' ports and cluster
-names as explained in the previous section. If you did, then you need to make certain changes to the network settings of your client.
+If you run the Hazelcast IMDG members on a different server than the client, you most probably have configured the members' ports and cluster names as explained in the previous section. If you did, then you need to make certain changes to the network settings of your client.
 
 #### 1.4.2.1. Cluster Name Setting
 
 You need to provide the name of the cluster, if it is defined on the server side, to which you want the client to connect.
 
-**Programmatic Configuration:**
-
 ```javascript
-let cfg = new Config.ClientConfig();
-cfg.clusterName = 'name of your cluster';
-```
-
-**Declarative Configuration:**
-
-```json
-{
-    "clusterName": "name of your cluster"
-}
+const cfg = {
+    clusterName: 'name of your cluster'
+};
 ```
 
 #### 1.4.2.2. Network Settings
 
 You need to provide the IP address and port of at least one member in your cluster, so the client can find it.
 
-**Programmatic Configuration:**
-
 ```javascript
-let cfg = new Config.ClientConfig();
-cfg.networkConfig.addresses.push('some-ip-address:port');
-```
-
-**Declarative Configuration:**
-
-```json
-{
-    "network": {
-        "clusterMembers": [
-            "some-ip-address:port"
+const cfg = {
+    networkConfig: {
+        addresses: [
```

Comment:
This is referred as `clusterMembers` in other parts

Reply:
I just saw that this problem exists in the `README.md`

Reply:
Good catch. Fixed in e3042e71bf4d3c6ff0db4b69a31ac871d5d6ae48